### PR TITLE
refactor: remove unnecessary string hashes

### PR DESCRIPTION
**Pull request by hamirmahal**
## Breaking Changes

This change is backward compatible.

## Changes

- [x] This removes unnecessary string hashes to improve readability.

### CODE REVIEW

**Review by hamirmahal — COMMENTED** [review]
@billy1624 are you open to this pull request?

File: tests/sqlite/query.rs [hunk 1]
```diff
@@ -910,12 +910,12 @@ fn select_55() {
...
+            r"ORDER BY CASE",
             r#"WHEN "id"=4 THEN 0"#,
             r#"WHEN "id"=5 THEN 1"#,
             r#"WHEN "id"=1 THEN 2"#,
             r#"WHEN "id"=3 THEN 3"#,
-            r#"ELSE 4 END,"#,
+            r"ELSE 4 END,",
             r#""glyph"."aspect" ASC"#,
         ]
         .join(" ")
```
Comment (tyt2y3) [inline tests/sqlite/query.rs:921]: Thank you for the above changes. 
While I think most removal are fine when they're clearly not necessary, this bits look inconsistent with the surrounding lines?

Can you revert the changes where it is in a vec of string and the other lines still need `r#`?

Comment (hamirmahal) [reply tests/sqlite/query.rs]: You're welcome. I reverted the changes in specific spots to make it more consistent.

File: tests/sqlite/table.rs
Comment (tyt2y3) [inline tests/sqlite/table.rs:1]: sorry I'd probably not change this file as well

**Comment by hamirmahal** [comment]
@tyt2y3 are you open to this change?

**Review by tyt2y3 — APPROVED** [review]
thank you

**Comment by hamirmahal** [comment]
You're welcome.
